packages/eslint-config-nimble-typescript/README.md

```diff
@@ -0,0 +1,63 @@
+# `@nimblehq/eslint-config-nimble-typescript`
```

Comment:
```suggestion
# @nimblehq/eslint-config-nimble-typescript
```

Reply:
Thanks for raising it!!! Nice catch 😛 

It was updated in cc65922 ✅ Please help to double-check it! 🙏🏻

Reply:
Seems you forgot to correct this one 🖌️

Reply:
It's consistent with other packages, that's why I choose to keep it as it currently is 😛